README.md

```diff
@@ -47,8 +47,18 @@ EOF
 $ terraform init
 ```
 
+When submitting PR make sure that it has updates to CHANGELOG.md. Add description of changes under last version with "(unreleased)" mark.
+
+Documenting provider
+--------------------
+Provider documentation is generated automatically when updates are merged into master branch.
```

Comment:
PR with renewed provider documentation is generated...

Reply:
Rewrote